### PR TITLE
 DBZ-7148: Blog post about JDBC Sink connector batch support

### PR DESCRIPTION
**Pull request by mfvitale**

### CODE REVIEW

File: _posts/2023-12-06-JDBC-sink-connector-batch-support.adoc [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+Therefore, the Debezium JDBC connector with batch support is 79 times faster than the version without batch support.
+
+.{nbsp}
+image::1M-different-batch-size-json.png[role=centered-image]
+
+In _Figure 4_, we observe the behavior of the Debezium JDBC connector writing on MySQL with different batch.size.
+Although the difference is notable at the beginning, it becomes apparent that the throughput continues to slow down, and, on average,
+all the `batch.size` configurations take about 7 minutes to process all events.
+
+This raised a concern for us. After conducting a thorough analysis (profiling), we identified that the bottleneck was in event deserialization.
```
Comment (vjuranek) [inline _posts/2023-12-06-JDBC-sink-connector-batch-support.adoc]: Does it really explains the difference between batches at the beginning? If deserialization is the root cause here, I'd expect that there won't be any difference also at the beginning. This looks like there are possibly also other factors which may affect it (my tip would be some kind of buffering somewhere).

If you have any explanation for it, I'd add it here.

**Review by vjuranek — COMMENTED** [review]
Nice write up - I had one minor comment, just in case you have more details about the differences. One general comment: I'd add units to the axis on all graphs, especially on the time axis.

File: _posts/2023-12-06-JDBC-sink-connector-batch-support.adoc (hunk 1 above)
Comment (mfvitale) [reply _posts/2023-12-06-JDBC-sink-connector-batch-support.adoc]: Good point. Maybe I think I should better clarify that this strange behavior lead us to investigate and we have found the problem with ser/der that solved the issue of batch.size scalability but not the slow down that can be related to other stuff that we have not investigated.

**Comment by mfvitale** [comment]
> Nice write up - I had one minor comment, just in case you have more details about the differences. One general comment: I'd add units to the axis on all graphs, especially on the time axis.

@vjuranek  Something like that? 
image

File: _posts/2023-12-06-JDBC-sink-connector-batch-support.adoc (hunk 1 above)
Comment (vjuranek) [reply _posts/2023-12-06-JDBC-sink-connector-batch-support.adoc]: yes, making this more clear would IMHO make the this part more understandable

**Comment by vjuranek** [comment]
> > Nice write up - I had one minor comment, just in case you have more details about the differences. One general comment: I'd add units to the axis on all graphs, especially on the time axis.
> 
> @vjuranek Something like that? 

yes, this would perfect

File: _posts/2023-12-06-JDBC-sink-connector-batch-support.adoc (hunk 1 above)
Comment (mfvitale) [reply _posts/2023-12-06-JDBC-sink-connector-batch-support.adoc]: @vjuranek  Do you think it's more clear now?

**Review by vjuranek — APPROVED** [review]
LGTM now!

**Comment by mfvitale** [comment]
@jpechane I would like to publish it before Xmas, do you think it's better to publish after the release announcement?

**Comment by jpechane** [comment]
@mfvitale IMHO it is independent so you can publish it even today. It is probably better to not publish two posts on the same date.

**Comment by mfvitale** [comment]
> @mfvitale IMHO it is independent so you can publish it even today. It is probably better to not publish two posts on the same date.

Ok, I'll update the date to today and then merge it. Thanks.

**Pull request merged**